main.go

```diff
@@ -22,6 +22,11 @@ import (
 	"google.golang.org/grpc"
 
 	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
+
+	"go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
```

Comment:
nit: grouping is [go standard library], [non-named imports], [named imports], [local]

Reply:
@dgl PTAL